src/main/java/mate.academy/Main.java

```diff
@@ -4,14 +4,16 @@
 import mate.academy.lib.Injector;
 import mate.academy.model.Product;
 import mate.academy.service.ProductService;
+import mate.academy.service.impl.ProductServiceImpl;
 
 public class Main {
 
     public static void main(String[] args) {
-        // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
-        List<Product> products = productService.getAllFromFile("products.txt");
+        ProductService productService =
+                (ProductService) injector.getInstance(ProductServiceImpl.class);
```

Comment:
Ensure that the `Injector` class has a `getInstance` method that correctly handles the instantiation of `ProductServiceImpl`. If this method is not implemented or not functioning as expected, it could lead to runtime errors.